### ISSUE TITLE
Trying to run openshift-apiserver scenario without success

### ISSUE THREAD

**Issue by kmlefebv** [cloud-bulldozer/kraken#85]
I've set up a standalone cluster for Kraken and followed the steps provided in the Readme to get started. I made some changes in the Kraken config file to only include one scenario which would be the openshift-apiserver to isolate the problem I'm having. This is what I have in the kraken deployment pod log after starting the deployment:
```
2021-04-12 14:16:22,450 [INFO] Starting kraken
2021-04-12 14:16:22,455 [INFO] Initializing client to talk to the Kubernetes cluster
/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py:1020: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.kraken-test-04.cp.fyre.ibm.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
2021-04-12 14:16:23,455 [INFO] Fetching cluster info
2021-04-12 14:16:47,718 [INFO]
I0412 14:16:24.777729      13 request.go:668] Waited for 1.168597466s due to client-side throttling, not priority and fairness, request: GET:https://api.kraken-test-04.cp.fyre.ibm.com:6443/apis/policy/v1beta1?timeout=32s
I0412 14:16:34.778149      13 request.go:668] Waited for 11.168923785s due to client-side throttling, not priority and fairness, request: GET:https://api.kraken-test-04.cp.fyre.ibm.com:6443/apis/coordination.k8s.io/v1beta1?timeout=32s
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.6.16    True        False         21h     Cluster version is 4.6.16
I0412 14:16:36.903365      31 request.go:668] Waited for 1.155303512s due to client-side throttling, not priority and fairness, request: GET:https://api.kraken-test-04.cp.fyre.ibm.com:6443/apis/networking.k8s.io/v1?timeout=32s
I0412 14:16:47.103691      31 request.go:668] Waited for 11.353934532s due to client-side throttling, not priority and fairness, request: GET:https://api.kraken-test-04.cp.fyre.ibm.com:6443/apis/admissionregistration.k8s.io/v1beta1?timeout=32s
Kubernetes control plane is running at https://api.kraken-test-04.cp.fyre.ibm.com:6443

2021-04-12 14:16:47,718 [INFO] Daemon mode enabled, kraken will cause chaos forever

2021-04-12 14:16:47,718 [INFO] Ignoring the iterations set
2021-04-12 14:16:47,718 [INFO] Executing scenarios for iteration 0
2021-04-12 14:16:48,211 [INFO] Scenario: scenarios/openshift-apiserver.yml has been successfully injected!
2021-04-12 14:16:48,212 [INFO] Waiting for the specified duration: 60
```

My expectation based on the message is that I would be able to see in real time a pod in the openshift-apiserver namespace be taken down, and a new one initialized. However, I don't see anything happening to the pods. This was with the default openshift-apiserver.yml scenario.
I've also tried changing the selector in the scenario to `app=openshift-apiserver-a` to match our targeted pod label in its yaml, as well as getting rid of the selector line altogether.
Are there additional steps that need to be done to get this scenario to work?

**Comment by bmos299** [comment]
Hi @paigerube14 can we get your thoughts on this one?

**Comment by paigerube14** [comment]
Are you able to post the config file you are using?

It seems like it's not trying to execute your scenario. Even if you are running the pod scenario and no pods match you would be getting output like the following; not what you are seeing so I am a little confused. 

`
....
2021-04-13 10:33:10,001 [INFO] Daemon mode not enabled, will run through 1 iterations

2021-04-13 10:33:10,001 [INFO] Executing scenarios for iteration 0
2021-04-13 10:33:14 INFO __main__ verbosity: None; log level: INFO; handler level: INFO
2021-04-13 10:33:14 INFO __main__ Creating kubernetes client with config /Users/prubenda/.kube/config from --kubeconfig flag
2021-04-13 10:33:14 INFO k8s_client Initializing with config: /Users/prubenda/.kube/config
2021-04-13 10:33:14 INFO __main__ No cloud driver - some functionality disabled
2021-04-13 10:33:14 INFO __main__ Using stdout metrics collector
2021-04-13 10:33:14 INFO __main__ NOT starting the UI server
2021-04-13 10:33:14 INFO __main__ STARTING AUTONOMOUS MODE
2021-04-13 10:33:17 INFO scenario.delete openshif Starting scenario 'delete openshift-apiserver pods' (2 steps)
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Matching 'labels' {'labels': {'namespace': 'openshift-apiserver', 'selector': 'app=openshift-apiserver'}}
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Matched 0 pods for selector app=openshift-apiserver in namespace openshift-apiserver
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Initial set length: 0
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Matching 'labels' {'labels': {'namespace': 'openshift-apiserver', 'selector': 'app=openshift-apiserver'}}
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Matched 0 pods for selector app=openshift-apiserver in namespace openshift-apiserver
2021-04-13 10:33:17 INFO action_nodes_pods.delete openshif Initial set length: 0
2021-04-13 10:33:17 INFO scenario.delete openshif Scenario finished
2021-04-13 10:33:17 INFO policy_runner All done here!
`

**Comment by kmlefebv** [comment]
Here's the Kraken `config.yaml` file:
```
kraken:
    kubeconfig_path: /root/.kube/config                    # Path to kubeconfig
    exit_on_failure: False                                 # Exit when a post action scenario fails
    litmus_version: v1.10.0                                # Litmus version to install
    litmus_uninstall: False                                # If you want to uninstall litmus if failure
    chaos_scenarios:                                       # List of policies/chaos scenarios to load
        -   pod_scenarios:                                 # List of chaos pod scenarios to load
            - -    scenarios/openshift-apiserver.yml

cerberus:
    cerberus_enabled: False                                # Enable it when cerberus is previously installed
    cerberus_url:                                          # When cerberus_enabled is set to True, provide the url where cerberus publishes go/no-go signal

performance_monitoring:
    deploy_dashboards: False                              # Install a mutable grafana and load the performance dashboards. Enable this only when running on OpenShift
    repo: "https://github.com/cloud-bulldozer/performance-dashboards.git"

tunings:
    wait_duration: 60                                      # Duration to wait between each chaos scenario
    iterations: 1                                          # Number of times to execute the scenarios
    daemon_mode: True                                     # Iterations are set to infinity which means that the kraken will cause chaos forever
```

I should mention that we received a similar output when we used the default `config.yaml` as well without the node scenarios .

Right now we are only using the `openshift-apiserver.yml` scenario and we are currently using the default version of that file. Are there any extra steps that aren't listed in the README KubeApp section that we need to perform?

**Comment by mffiedler** [comment]
Fixed by #86
